test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/npd-binding.yaml

```diff
@@ -0,0 +1,16 @@
+# This is the role binding for the node-problem-detector
+# which is used to get and patch the node health status.
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: node-problem-detector-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
The node problem detector is a pretty "normal" add-on, right?  Do you have a list of rbac denies for this user?  I could see making a role for it.